pkg/cloud-provider/cloudapi/aws/aws_security.go

```diff
@@ -204,8 +205,12 @@ func (ec2Cfg *ec2ServiceConfig) realizeIngressIPPermissions(cloudSgObj *ec2.Secu
 		if rule == nil {
 			continue
 		}
-		idGroupPairs := buildEc2UserIDGroupPairs(rule.FromSecurityGroups, cloudSGNameToObj)
-		ipRanges := convertToEc2IpRanges(rule.FromSrcIP, len(rule.FromSecurityGroups) > 0)
+		description, err := securitygroup.GenerateCloudDescription(obj.NetworkPolicy, *cloudSgObj.GroupName)
+		if err != nil {
+			return fmt.Errorf("unable to generate rule description, err: %v", err)
```

Comment:
this shouldnt happen right for all practical purpose?

Reply:
Yes.